### PR TITLE
Update the RELEASE instructions

### PR DESCRIPTION
**Pull request by jdfreder**
@jasongrout @SylvainCorlay so we can remember.

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -1,9 +1,15 @@
 To release a new version of the widgets on PyPI:
 
-1. Update \_version.py
+1. Update \_version.py (set release version, remove 'dev')
 2. `git add` and `git commit`
 3. `python setup.py sdist upload` 
 4. `python setup.py bdist_wheel upload`
```
Comment (SylvainCorlay) [inline RELEASE.md]: Is it ok when it is on two different lines?

Comment (jdfreder) [inline RELEASE.md]: It's what I've been doing.  Is there a better way to do it?  I'm a PyPI noob

Comment (blink1073) [inline RELEASE.md]: Yeah, you can upload as many times as you want (as long as it is a different file)

Comment (SylvainCorlay) [inline RELEASE.md]: Me too, just that it is different from the wheels tutorial, hence the question. If it worked, you answered my question :)

**Pull request merged**